src/libs/API.js

```diff
@@ -327,6 +327,16 @@ function AuthenticateWithAccountID(parameters) {
     });
 }
 
+/**
+ * @param {Object} parameters
+ * @returns {Promise}
+ */
+function AddBillingCard(parameters) {
+    const commandName = 'User_AddBillingCard';
+    return Network.post(commandName, parameters);
```

Comment:
This line needs to be `return Network.post(commandName, parameters, CONST.NETWORK.METHOD.POST, true);` because the command is on our secure server, not the normal one.